doc/source/Resources/pyaedt_settings.yaml

```diff
@@ -0,0 +1,117 @@
+# This file contains the settings used by default to set the PyAEDT environment variables and
+# global settings. If you want to have a different behavior you have to copy / paste this file
+# into your PersonalLib directory and restart your workflow.
+
+# Settings related to logging
+log:
+  # Enable or disable the logging of EDB API methods
+  enable_debug_edb_logger: false
+  # Enable or disable the logging of the geometry operators
+  enable_debug_geometry_operator_logger: false
+  # Enable or disable the logging of the gRPC API calls
+  enable_debug_grpc_api_logger: false
+  # Enable or disable the logging of internal methods
+  enable_debug_internal_methods_logger: false
+  # Enable or disable the logging at debug level
+  enable_debug_logger: false
+  # Enable or disable the logging of methods' arguments at debug level
+  enable_debug_methods_argument_logger: false
+  # Enable or disable the logging to the AEDT message window
+  enable_desktop_logs: true
+  # Enable or disable the logging to a file
+  enable_file_logs: true
+  # Enable or disable the global PyAEDT log file located in the global temp folder
+  enable_global_log_file: true
+  # Enable or disable the local PyAEDT log file located in the ``projectname.pyaedt`` project folder
+  enable_local_log_file: false
+  # Enable or disable the logging overall
+  enable_logger: true
+  # Enable or disable the logging to STDOUT
+  enable_screen_logs: true
+  # Global PyAEDT log file path
+  global_log_file_name: null
+  # Global PyAEDT log file size in MB
+  global_log_file_size: 10
+  # Date format of the log entries
+  logger_datefmt: '%Y/%m/%d %H.%M.%S'
+  # PyAEDT log file path
+  logger_file_path: null
+  # Message format of the log entries
+  logger_formatter: '%(asctime)s:%(destination)s:%(extra)s%(levelname)-8s:%(message)s'
+
+# Settings related to Linux systems running LSF scheduler
+lsf:
+  # Command to launch in the LSF Scheduler
+  custom_lsf_command: null
+  # Command to launch the task in the LSF Scheduler
+  lsf_aedt_command: 'ansysedt'
+  # Number of LSF cores
+  lsf_num_cores: 2
+  # Operating system string
+  lsf_osrel: null
+  # LSF queue name
+  lsf_queue: null
+  # RAM allocated for the LSF job
+  lsf_ram: 1000
+  # Timeout in seconds for trying to start the interactive session
+  lsf_timeout: 3600
+  # Value passed in the LSF 'select' string to the ui resource
+  lsf_ui: null
+  # Enable or disable use LSF Scheduler
+  use_lsf_scheduler: false
+
+# Settings related to environment variables thare are set before launching a new AEDT session
```

Comment:
```suggestion
# Settings related to environment variables are set before launching a new AEDT session
```

Reply:
I wanted to specify that the following settings are needed before launching AEDT. I can switch from `"that"` to `"which"` but I feel like you miss a part from the message by removing any of them.